code/modules/guardian/abilities/special/absolution.dm

```diff
@@ -0,0 +1,160 @@
+/datum/guardian_ability/major/special/onewayroad // a cookie for you if you get the reference
```

Comment:
:sadface: nobody is gonna get the reference

Reply:
tru